### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by ionut-arm**

### CODE REVIEW

**Review by Superhepper — COMMENTED** [review]
Should we not be doing the bindgen update and regenerating the bindings in sys first?

**Comment by ionut-arm** [comment]
ah, yes, as per #386. I'll port that to v7 as well.

cc @aplanas

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [tss-esapi-7.2.0](https://github.com/parallaxsecond/rust-tss-esapi/tree/tss-esapi-7.2.0) (2023-02-02)
+## [tss-esapi-sys-0.3.1](https://github.com/parallaxsecond/rust-tss-esapi/tree/tss-esapi-sys-0.3.1) (2023-02-02)
+
+[Full Changelog](https://github.com/parallaxsecond/rust-tss-esapi/compare/tss-esapi-7.1.0...tss-esapi-sys-0.3.1)
```
Comment (aplanas) [inline CHANGELOG.md:6]: maybe https://github.com/parallaxsecond/rust-tss-esapi/compare/tss-esapi-7.2.0...tss-esapi-7.1.0 ?

**Review by wiktor-k — APPROVED** [review]
Looks good~! :tada:

**Comment by ionut-arm** [comment]
I've decided to bump the `tss-esapi-sys` version from `0.3.0` to `0.4.0` following the bindgen changes.

**Review by Superhepper — APPROVED** [review]

**Pull request merged**